adala/runtimes/_litellm.py

```diff
@@ -123,7 +120,40 @@ def _get_usage_dict(usage: Usage, model: str) -> Dict:
     return data
 
 
-class LiteLLMChatRuntime(Runtime):
+class InstructorClientMixin:
+    client: Any = Field(default=None, exclude=True)
+
+    def _from_litellm(self, **kwargs):
+        return instructor.from_litellm(litellm.completion, **kwargs)
+
+    def use_instructor_client(self) -> None:
+        if self.client is None:
+            kwargs = {}
+            if self.is_custom_openai_endpoint:
+                kwargs["mode"] = instructor.Mode.JSON
```

Comment:
this is the kind of thing that should go in ModelParams later and be passed into the runtime, right? mind adding a comment about it?

Reply:
yes, comment added